test/e2e/apimachinery/aggregator.go

```diff
@@ -159,7 +161,9 @@ func setupSampleAPIServerCert(namespaceName, serviceName string) *aggregatorCont
 func TestSampleAPIServer(f *framework.Framework, image, namespaceName string) {
 	By("Registering the sample API server.")
 	cleanTest(f, true)
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Better not to seed the time? I think go seeds by default. You can use the crypto random number package if you are worried.

Reply:
It does not.
package main

import (
	"fmt"
	"math/rand"
)

func main() {
	fmt.Printf("Hello, playground %d", rand.Int())
}

always generates "Hello, playground 134020434" for me.

Reply:
Hm. Still, there should be a bunch of calls to rand in our test binaries already, to generate the namespace names if nothing else.